adapter/src/debug_session/variables.rs

```diff
@@ -372,7 +372,7 @@ impl super::DebugSession {
     fn get_container_summary(var: &SBValue) -> String {
         // Summary evaluation is supposed to be quick, so cap it at 10ms.
         const MAX_EVALUATION_TIME: time::Duration = time::Duration::from_millis(10);
-        const MAX_LENGTH: usize = 32;
+        const MAX_LENGTH: usize = 512;
```

Comment:
This code is optimized for speed, as it is primarily used for rendering the variables side panel, which is rarely wide enough to display more that 32 chars.  Since you want to get each item's summary (which may call out to Python code), I'm kinda concerned about performance in the more extreme cases (say, when there's an array of arrays)...

Reply:
Makes sense! Would the PR be interesting for you when I remove the commit that changes `MAX_LENGTH`?

Reply:
Yes, that would be fine I think.

Reply:
ty! I updated the PR (removed the `MAX_LENGTH` commit)